JSTests/stress/big-int-spec-to-this.js

```diff
@@ -18,6 +17,3 @@ for (let i = 0; i < 10000; i++) {
 for (let i = 0; i < 10000; i++) {
     assert(foo.apply(300) === "object");
 }
-
-assert(numberOfDFGCompiles(foo) === 1);
-
```

Comment:
Can you explain why this is assert is invalid?  Why did you remove it?

Reply:
Oh I guess I was seeing flaky results locally because I missed the `useConcurrentJIT=0`. Brought back.